tests/consensus_specs_test.go

```diff
@@ -49,6 +49,8 @@ func commonPrefix(a []byte, b []byte) []byte {
 // consensus spec tests repo and runs the encoding/decoding/hashing round.
 func TestConsensusSpecBasics(t *testing.T) {
 	testConsensusSpecBasicType[*types.SingleFieldTestStruct](t, "SingleFieldTestStruct")
+	testConsensusSpecBasicType[*types.SingleFieldTestStruct](t, "SmallTestStruct")
+	testConsensusSpecBasicType[*types.SingleFieldTestStruct](t, "FixedTestStruct")
```

Comment:
Heh, you need to change the type name too :P `types.SingleFieldTestStruct` -> your two new types :)

Reply:
really, sorry for this 
thank you for not getting annoyed on so so silly issues and guiding in a detailed manner

Reply:
It's ok :) You're sacrificing your time on this :)